generate.py

```diff
@@ -525,3 +530,108 @@ def normalize(code):
 { content }
 ''')
 
+### Generate Quality Estimation files
+
+for estimation in QUALITY_ESTIMATION:
+
+  name = estimation['name']
+  if not isinstance(name, str):
+    raise Exception(name)
+
+  company_id = estimation['id']
+  if not isinstance(company_id, str):
+    raise Exception(company_id)
+  
+  company = estimation['company'].title().replace('-',' ')
+  if not isinstance(company, str):
+    raise Exception(company)
+  
+  description = estimation['description']
+  if not isinstance(description, str):
+    raise Exception(description)
+  
+  languages = estimation['languages']
+  if not isinstance(languages, list):
+    raise Exception(languages)
+
+  urls = estimation['urls']
+  if not isinstance(urls, list):
+    raise Exception(urls)
+
+  only_compatible_mt_api = []
+  only_compatible_mt_api_id = estimation.get('only_compatible_mt_api', False)
+  api_name = next((api['name'] for api in APIS if only_compatible_mt_api_id and \
+                   api['id'].lower() == only_compatible_mt_api_id.lower()), only_compatible_mt_api_id)
+  only_compatible_mt_api.append({
+    'id': only_compatible_mt_api_id,
+    'name': api_name
+  })
+
+  only_compatible_tms = []
+  only_compatible_tms_id = estimation.get('only_compatible_tms', False)
+  tms_name = next((tms['name'] for tms in INTEGRATIONS if only_compatible_tms_id and \
+                   tms['id'].lower() == only_compatible_tms_id.lower()), only_compatible_tms_id)
+  only_compatible_tms.append({
+    'id': only_compatible_tms_id,
+    'name': tms_name
+  })
+
+  customisation = estimation.get('customisation', False)
+
+  supported_language_codes = list(set(flatten(languages)))
+  supported_language_codes.sort()
+  
+  supported_languages = []
+  for code in supported_language_codes:
+    language_name = None
+    language_slug = None
+    for lang in LANGUAGES:
+      normalized_code = normalize_language_code(code, company_id)
+      base_code = base_language_code(normalized_code)
+      if base_code in lang['codes']:
+        language_name = lang['names'][0]
+        language_slug = slugify(language_name)
+        break
+    variant_name = get_language_variant_name(code, company_id)
+    supported_languages.append({
+      'slug': language_slug,
+      'code': code,
+      'base_code': base_code,
+      'name': language_name,
+      'variant_name': variant_name
+    })
+
+  desc = f'{description}'
```

Comment:
This is pointless, so we don't need `desc`.

Reply:
`description` is a marketing tagline from the provider.

Here we should use something more standard, like we do for the machine translation APIs.
```
desc = f'The { title } translation quality estimation API'
```

Then we should change `description` in quality-estimation.json to `tagline`.

And the tagline should just be in frontmatter as `'tagline': tagline`, and then in the article it can be in a blockquote.

Eventually we could do similar for APIs and aggregators too, but the problem is that they change and/or are not even consistent for one company.

So if we just decide to drop taglines completely, that's fine too.